redaxo/src/core/lib/console/list_command.php

```diff
@@ -13,7 +13,7 @@
  */
 class rex_command_list extends ListCommand
 {
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
Den Command musste ich anpassen, weil wir da den nativen ListCommand extenden, und der hat neu den Return-type.
Aber in der Basis-Command-Klasse ist der Return-Type nicht notiert (nur als phpdoc). Daher müssen alle normalen Commands ihn nicht hinzufügen.